packages/paste-website/src/components/DoDont.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Text} from '@twilio-paste/text';
+
+const StyledWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  grid-column-gap: 24px;
+`;
+
+interface DoDontProps {
+  children?: React.ReactNode;
+}
+
+const DoDont: React.FC<DoDontProps> = props => {
+  return <StyledWrapper>{props.children}</StyledWrapper>;
+};
+
+const StyledExampleImg: React.FC<ExampleProps> = styled.div`
+  margin-bottom: ${themeGet('space.space50')};
+  border: ${themeGet('borderWidths.borderWidth10')} solid #ccd2dc;
+  border-bottom: ${(props: ExampleProps) =>
+    props.do
+      ? `${themeGet('borderWidths.borderWidth20')(props)} solid #23bf6e`
+      : `${themeGet('borderWidths.borderWidth20')(props)} solid #ce241a`};
+  img {
+    display: block;
+    width: 100%;
+  }
+`;
+
+interface ExampleProps {
```

Comment:
nit: could you move this above `StyledExampleImg` since that functional component uses this interface please? easier to read that way and sometimes JS throws errors when you define something after it's used, so it's best to be consistent I think